src/header_util.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this supposed to be committed?

Reply:
Ugh.  Guessing that this got created, then deleted, then somehow managed to recreate itself during a hairy rebase.  Thanks for the catch.